setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 160
```

Comment:
this value is too large, either stick to 79 or 120
also provide this to black
additionally, you should add isort to the quality checks and set its profile to black and specify the max length